src/main/recources/ActionPerDay.csv

```diff
@@ -0,0 +1,22 @@
+type,fruit,quantity

```

Comment:
directory resources must be system main -> new -> directory 
<img width="330" alt="image" src="https://user-images.githubusercontent.com/96740815/175042633-df18abc3-a2d5-4190-b095-70e99e964c44.png">